presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -120,6 +133,14 @@ public AggregationNode(@JsonProperty("id") PlanNodeId id,
         return symbols.build();
     }
 
+    public List<Symbol> getAllGroupingColumns()
```

Comment:
most likely all places where you call `isEmpty` on the result of this function is wrong